client/cluster-client/src/main/kotlin/uk/ac/ebi/biostd/client/cluster/api/RemoteClusterClient.kt

```diff
@@ -49,6 +49,13 @@ class RemoteClusterClient(
         }
     }
 
+    override suspend fun jobLogs(jobId: String): String {
+        return runInSession {
+            val (_, response) = executeCommand("cat $logsPath/${jobId}_OUT")
```

Comment:
And there we were, thinking about datamovers and separate jobs 🤣 

Well done man 👌